src/app/components/sections/footer.tsx

```diff
@@ -22,26 +26,103 @@ const defaultFAQs = [
+                accordionRootClassName="w-full lg:w-[624px]"
+              />
+            </div>
+          </div>
+        )}
+
+        {/* Footer Links Section */}
+        <div className={`grid grid-cols-1 md:grid-cols-3 gap-10 md:gap-20 text-white ${
+          showFAQSection ? 'border-t border-grey-accordion-background pt-10 md:pt-20' : ''
+        } px-6 md:px-0`}>
+          <div className="flex flex-col gap-6 relative md:pl-10">
+            <h3 className="text-xl font-semibold">Contact</h3>
+            <div className="flex flex-col gap-4">
+              <Link href="https://twitter.com" className="text-grey-footer-text hover:text-white">Twitter</Link>
+              <Link href="https://discord.com/channels/1189471179967315968/1275042830028505088" className="text-grey-footer-text hover:text-white" target="_blank" rel="noopener noreferrer">Discord</Link>
+            </div>
+            <div className="hidden md:block absolute right-0 top-0 h-full w-[1px] bg-grey-accordion-background"></div>
+          </div>
+
+          <div className="flex flex-col gap-6 relative">
+            <h3 className="text-xl font-semibold">Menu</h3>
+            <div className="flex flex-col gap-4">
+              <Link 
+                href="/#features" 
+                className="text-grey-footer-text hover:text-white"
+                scroll={true}
+              >
+                Our Features
+              </Link>
+              <Link 
+                href="/#curriculum" 
+                className="text-grey-footer-text hover:text-white"
+                scroll={true}
+              >
+                Our Curriculum
+              </Link>
+            </div>
+            <div className="hidden md:block absolute right-0 top-0 h-full w-[1px] bg-grey-accordion-background"></div>
+          </div>
+
+          <div className="flex flex-col gap-6">
+            <h3 className="text-xl font-semibold">Resources</h3>
+            <div className="flex flex-col gap-4">
+              <Link href="/faqs" className="text-grey-footer-text hover:text-white">FAQs</Link>
+              {/* TODO: @extheo should confirm what contact is */}
+              {/* <Link href="/contact" className="text-grey-footer-text hover:text-white">Contact Us</Link> */}
+            </div>
+          </div>
+        </div>
+
+        {/* Footer Bottom */}
+        <div className="flex flex-col gap-4 border-t border-grey-accordion-background pt-10 mt-10">
+          <div className="px-6 md:pl-10 -ml-6">
+            <Link href="/">
+              <Image 
+                src="/assets/images/logo.svg"
+                alt="Logo"
+                width={116}
+                height={88}
+                priority
+              />
             </Link>
           </div>
-          <div className="flex flex-col items-left gap-4 text-white">
-            <AccordionComponent data={faqData} />
+          <p className="text-grey-footer-text max-w-[500px] px-6 md:pl-10 text-sm md:text-base">
```

Comment:
This font should be inter. I think removing the p22-Mackinac font you set at the parent component in the site layout should do the trick and automatically assign Inter font as it's the default. Then you can assign the p22-Mackinac to the specific sections that needs it

Reply:
fixed - please check.